config/environments/production.rb

```diff
@@ -62,7 +62,7 @@
   # Use a real queuing backend for Active Job
   config.active_job.queue_adapter = :resque
 
-  config.action_mailer.default_url_options = { host: 'sul-preassembly-prod.stanford.edu', protocol: 'https' } # needed by url_for() in mail templates
+  config.action_mailer.default_url_options = { host: Settings.mailer_host || 'sul-preassembly-prod.stanford.edu', protocol: 'https' } # needed by url_for() in mail templates
```

Comment:
what about always using `Settings.mailer_host` instead of the `||` logic here, and adding a default to the base `settings.yml`?

Reply:
i guess if you did it all through settings, i'd also propose using a dummy value for the base `settings.yml` config, and then putting the prod URL in the prod shared_configs. but similarly, not something i feel strongly about for this usage.  but that feels like the slightly more intuitive/consistent implementation to me.

Reply:
Yup, that's the approach I took. Updated.